spec-main/chromium-spec.ts

```diff
@@ -476,37 +476,6 @@ describe('chromium features', () => {
       expect(preferences.javascript).to.be.false()
     })
 
-    it('handles cycles when merging the parent options into the child options', (done) => {
```

Comment:
I don't think this was a necessary requirement? Happy to be shown that I'm wrong and reinstate the functionality, but I don't see how cycle prevention is helpful.